manifests/d/Datadog/Agent/7.28.1/Datadog.Agent.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: Datadog.Agent
+PackageVersion: 7.28.1

```

Comment:
```suggestion
PackageVersion: 7.28.1.1
```
This is what I'm seeing from Control Panel - updating this to match the one shown in Control Panel allows for the best upgrading experience when using the `winget upgrade` command.
![image](https://user-images.githubusercontent.com/11600822/120695900-ccd29c00-c4a3-11eb-998e-048b0aa3b0bb.png)